pkg/compute/serve.go

```diff
@@ -52,6 +53,7 @@ func NewServeCommand(parent cmd.Registerer, globals *config.Data, build *BuildCo
 
 	c.CmdClause.Flag("env", "The environment configuration to use (e.g. stage)").Action(c.env.Set).StringVar(&c.env.Value)
 	c.CmdClause.Flag("file", "The Wasm file to run").Default("bin/main.wasm").StringVar(&c.file)
+	c.CmdClause.Flag("addr", "The IPv4 address to listen on").Default("127.0.0.1:7676").StringVar(&c.addr)
```

Comment:
Do we really want to couple the CLI to Viceroy like this? If there's a new release of Viceroy that uses a different port for example, then it means we'd need to release a new CLI version. Might be best to only set the `--addr` flag if a value is provided rather than providing a default.

Reply:
Yeah I did spend some time thinking about this. As we always pass this arg (with default value or user provded one) Its better to think about it that we're explicitly controlling the default value from the CLI and that is independent over whatever Viceroy decides to use as its default. I.e. if Viceroy updates their default, the CLI doesn't need to change as its always passing 7878 explicitly.

Per my comment on your PR, I "personally" prefer to use flag defaults as it then generates the documentation so users can discover what the default is rather than having magic defaults.

Reply:
> Its better to think about it that we're explicitly controlling the default value from the CLI and that value is independent from whatever Viceroy decides to use as its default. I.e. if Viceroy updates their default, the CLI doesn't need to change as its always passing 7878 explicitly.

OK this makes sense 👍🏻